### PR TITLE
Chinese Remainder Theorem added

### PR DESCRIPTION
**Pull request by muddhit**
### Your checklist for this pull request
:warning: Please review the [guidelines for contributing](https://github.com/sanscript-tech/hacking-tools-scripts/blob/main/CONTRIBUTING.md) to this repository.

Parent issue: #33 

### Type of change

**Select all that applies**:
* [x] New feature (non-breaking change which adds functionality)
* [ ] Bug fix (non-breaking change which fixes an issue)
* [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
* [ ] This change requires a documentation update

### Checklist:

* [x] My code follows the style guidelines(Clean Code) of this project.
* [x] I have performed a self-review of my own code.
* [x] I have used consistent code conventions and clear function/method/variable names.
* [x] I have commented on my code, particularly in hard-to-understand areas.
* [x] I have created a helpful and easy to understand README.md which contains description, requirements and use case.
* [x] I have added relevent screenshots/images if necessary in a seperate folder.
* [x] I have added my work in respective language folder.
* [x] My changes generate no new warnings.

### CODE REVIEW

**Comment by muddhit** [comment]
I have added README file in the next pull request because I forgot to add it in this one,Sorry for the inconvenience.

**Review by yashaswibiyahut — CHANGES_REQUESTED** [review]
Add readme file in this PR

**Comment by muddhit** [comment]
@yashaswibiyahut required changes have been committed.

**Comment by yashaswibiyahut** [comment]
@Mukulbaid63 please add a running code

**Comment by yashaswibiyahut** [comment]
kindly add a dynamic input and also add time and space complexity rest LGTM.

**Review by muddhit — COMMENTED** [review]
Added Time complexity and Dynamic input feature @yashaswibiyahut

**Comment by yashaswibiyahut** [comment]
(Java) _ Online editor and compiler - Google Chrome 10_28_2020 11_55_51 AM (2)
**Add Running Code**. @Mukulbaid63

**Review by muddhit — COMMENTED** [review]
Sorry for the inconvenience,added the missing code.Please review @yashaswibiyahut

File: Java/ChineseRemainderTheorem/CRT.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	
+	} 
+	
+	// Driver method 
+	public static void main(String args[]) 
+	{ /*Given two arrays div and rem,we need to find out the minimum number 'A' */
+    /*,for which when the number of div array is divided it gives the corresponding value present in rem array*/
+		//Taking input the length of the div array
+		Scanner sc =new Scanner(System.in);
+                int n=sc.nextInt();
```
Comment (yashaswibiyahut) [inline Java/ChineseRemainderTheorem/CRT.java:40]: proper line alignment.

**Review by yashaswibiyahut — CHANGES_REQUESTED** [review]
Small changes required 
Rest LGTM

**Review by muddhit — COMMENTED** [review]
Aligned. @yashaswibiyahut

**Review by yashaswibiyahut — APPROVED** [review]

**Pull request merged**